### PR TITLE
x11-misc/shutter: version 0.96

### PR DESCRIPTION
**Pull request by DarthGandalf**

### CODE REVIEW

File: dev-perl/GooCanvas2-CairoTypes/GooCanvas2-CairoTypes-0.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+DESCRIPTION="Bridge between GooCanvas2 and Cairo types"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="
+	x11-libs/goocanvas:2.0[introspection]
+	dev-perl/Cairo
```
Comment (akhuettel) [inline dev-perl/GooCanvas2-CairoTypes/GooCanvas2-CairoTypes-0.1.0.ebuild:18]: Missing Gtk3
https://metacpan.org/source/ASOKOLOV/GooCanvas2-CairoTypes-0.001/META.json#L52

File: dev-perl/Gtk3-ImageView/Gtk3-ImageView-9.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DIST_AUTHOR=ASOKOLOV
+DIST_VERSION=9
+DIST_TEST=do
```
Comment (akhuettel) [inline dev-perl/Gtk3-ImageView/Gtk3-ImageView-9.0.0.ebuild:8]: Not needed unless you want to explicitly specify that the tests should run in sequence (i.e. not parallelized).
The default is DIST_TEST="do parallel"

File: x11-misc/shutter/shutter-0.96.ebuild [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+DESCRIPTION="Feature-rich screenshot program"
+HOMEPAGE="https://shutter-project.org/"
+SRC_URI="https://github.com/shutter-project/shutter/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	dev-lang/perl
```
Comment (akhuettel) [inline x11-misc/shutter/shutter-0.96.ebuild:17]: Check if it installs Perl modules (into perl5/vendor_perl/5.xx/) or libraries linked to libperl.so. If yes, then add := operator...

**Review by akhuettel — APPROVED** [review]
Looks good, needs only minor changes.

File: x11-misc/shutter/shutter-0.96.ebuild (hunk 3 above)
Comment (DarthGandalf) [reply x11-misc/shutter/shutter-0.96.ebuild:17]: It does not

File: dev-perl/Gtk3-ImageView/Gtk3-ImageView-9.0.0.ebuild (hunk 2 above)
Comment (DarthGandalf) [reply dev-perl/Gtk3-ImageView/Gtk3-ImageView-9.0.0.ebuild:8]: Yes, it's broken when in parallel, so I'm disabling parallelism

File: dev-perl/GooCanvas2-CairoTypes/GooCanvas2-CairoTypes-0.1.0.ebuild (hunk 1 above)
Comment (DarthGandalf) [reply dev-perl/GooCanvas2-CairoTypes/GooCanvas2-CairoTypes-0.1.0.ebuild:19]: Fixed

**Pull request closed without merging**